src/utils.js

```diff
@@ -149,23 +150,48 @@ const allowedMarkups = [
   'delete',
 ];
 
+const matchMarkdownLinks = (message) => {
+  const regexMdLinks = /\[([^\[]+)\](\(.*\))/gm;
+  const matches = message.match(regexMdLinks);
+  // console.log('links', matches);
+
+  const singleMatch = /\[([^\[]+)\]\((.*)\)/;
+  let links = [];
```

Comment:
Minor, but we could go functional and replace the `for` loop for `matches.map(match => singleMatch.exec(match)[2]);` (didn't test) here.

Reply:
good one, pushed that.